wled00/bus_manager.cpp

```diff
@@ -1028,8 +1014,8 @@ void __attribute__((hot)) BusHub75Matrix::setPixelColor(unsigned pix, uint32_t c
     uint8_t g = G(c);
     uint8_t b = B(c);
 
-    if(virtualDisp != nullptr) {
-      int x = pix % _panelWidth;
+    if (virtualDisp != nullptr) {
+      int x = pix % _panelWidth; // TODO: check if using & and shift would be faster here, it limits to power-of-2 widths though
```

Comment:
comment to your comment: this is in the fallback code, so any optimization will only affect users with "low RAM" problems, like esp32-s2.
"extensa" based chips (esp32, -S3, -S2) have integer division in hardware, which needs ~2 CPU cycles. I'm not sure if "modulo" is a natural result of the division (divmod?), but I'd expect that any optimization here will only win 1 or 2 CPU cycles ... as you say, at the price of dropping support for non-power-of-two panel sizes. Especially for 80x40 which are quite common on ali.